seeker-tiny.zsh-theme

```diff
@@ -18,7 +18,7 @@ function _pyenv_prompt_info {
         echo "${ZSH_THEME_PYENV_PROMPT_PREFIX}$(pyenv_prompt_info)${ZSH_THEME_PYENV_PROMPT_SUFFIX}"
 }
 
-PROMPT=' %{$fg_bold[green]%}%n@%M:%{$fg[cyan]%}%~ %{$fg_bold[blue]%}$(_virtualenv_prompt_info)$(_pyenv_prompt_info)$(_git_prompt_info)$(_hg_prompt_info)%{$fg_bold[blue]%}% 
+PROMPT=' %{$fg_bold[green]%}%n@%M:%{$fg[cyan]%}%~ %{$fg_bold[blue]%}$(virtualenv_prompt_info)$(_pyenv_prompt_info)$(_git_prompt_info)$(_hg_prompt_info)%{$fg_bold[blue]%}% 
```

Comment:
`_virtualenv_prompt_info` is defined in upside. Using `virtualenv_prompt_info` directly will cause errors while `oh-my-zsh-virtualenv-prompt` not installed.